crates/pyext/Cargo.toml

```diff
@@ -28,3 +28,4 @@ qiskit-accelerate.workspace = true
 qiskit-circuit.workspace = true
 qiskit-qasm2.workspace = true
 qiskit-qasm3.workspace = true
+qiskit-cext = { workspace = true, features = ["python_binding"] }
```

Comment:
This change made me look again at the Makefile, and I realised there's a couple of mistakes in it in the C build.  I can push them up to this PR, if it's easier?

Reply:
Yeah feel free to push those up to this PR directly that's fine.

Reply:
Done in b25868cd7f